frontend/src/components/core/Sidebar/Sidebar.tsx

```diff
@@ -121,6 +157,23 @@ class Sidebar extends PureComponent<SidebarProps, State> {
     }
   }
 
+  static shouldNavCollapse(
+    pageQuantity: number,
+    hasElements: boolean,
+    collapsedPreference: boolean
```

Comment:
I think the `collapsedPreference` param should be named `expandedPreference` based on how this function's logic and use. In both calls to `shouldNavCollapse` this input is `cachedSidebarNavExpandedPreference` (expand true, collapse false).

Reply:
Makes sense! Fixed on [91e9a74](https://github.com/streamlit/streamlit/pull/6050/commits/91e9a7441e3a44d625374a6537b965b0ea952228)